src/gui/optionsdlg.cpp

```diff
@@ -1182,6 +1206,9 @@ bool OptionsDialog::useAdditionDialog() const
 void OptionsDialog::enableApplyButton()
 {
     applyButton->setEnabled(true);
+
+    // Verify if the share action button must be enabled
+    m_ui->comboRatioLimitAct->setEnabled(m_ui->checkMaxRatio->isChecked() || m_ui->checkMaxSeedingMinutes->isChecked());
```

Comment:
:) That's because you haven't tried the PR, just read the code.  If neither the option "Seed torrents until their ratio reaches" nor  "Seed torrents until their seeding time reaches" are enabled, the action (where you select if you want to Pause or Remove the torrent) should be disabled.  It's the same functionality than before but now you have to select either one of the options.  

Originally if you select "Seed torrents until their ratio reaches" enables the Action drop-down field and if you unselect it, it disables it.  Same thing but now with two options.

I welcome to try the PR.  It works great and it's a really useful feature :)